jax/interpreters/masking.py

```diff
@@ -239,6 +245,12 @@ def evaluate(self, env):
   def is_constant(self):
     return len(self) == 1 and next(iter(self)).degree == 0
 
+  @property
+  def _leading_term(self) -> Tuple['Mon', int]:
```

Comment:
What is the significance of the leading_term? Is it the constant if one is present?

Reply:
It's needed by polynomial division to determine the next term to divide.